package/MDAnalysis/guesser/default_guesser.py

```diff
@@ -0,0 +1,601 @@
+    * bonds
+    * improper dihedrals
+    * aromaticities
+
+    You can use this guesser either directly through an instance, or through the 
+    :meth:`~MDAnalysis.core.universe.Universe.guess_TopologyAttributes` method.
+
+    Examples
+    --------
+    to guess bonds for a universe::
+
+        import MDAnalysis as mda
+        from MDAnalysisTests.datafiles import two_water_gro
+
+        u = mda.Universe(two_water_gro, context = 'default', to_guess=['bonds'])
+
+    .. versionadded:: 2.7.0
+
+    """
+    context = 'default'
+
+    def __init__(self, universe, **kwargs):
+        super().__init__(universe, **kwargs)
+        self._guesser_methods = {
+            'masses': self.guess_masses,
+            'types': self.guess_types,
+            'elements': self.guess_types,
+            'bonds': self.guess_bonds,
+            'angles': self.guess_angles,
+            'dihedrals': self.guess_dihedrals,
+            'impropers': self.guess_improper_dihedrals,
+            'aromaticities': self.guess_aromaticities,
+        }
+
+    def guess_masses(self, atom_types=None, indices_to_guess=None):
+        """Guess the mass of many atoms based upon their type.
+        For guessing masses through Univese.guess_topologyAttribute():
+        First it try to guess masses from atom elements, if not available,
+        try to guess masses from types and if not availaible, try to guess
+        types.
+
+        Parameters
+        ----------
+        atom_types
+          Atom types/elements to guess masses from
+        indices_to_guess (optional)
+          Mask array for partially guess masses for certain atoms
+
+        Returns
+        -------
+        atom_masses : np.ndarray dtype float64
+
+        Raises
+        ------
+        :exc:`ValueError` if there is no atom types
+         or elements to guess mass from.
+
+        """
+        if atom_types is None:
+            try:
+                atom_types = self._universe.atoms.elements
+            except AttributeError:
+                try:
+                    atom_types = self._universe.atoms.types
+                except AttributeError:
+                    try:
+                        atom_types = self.guess_types(atom_types=self._universe.atoms.names)
+                    except ValueError:
+                        raise ValueError(
+                            "there is no reference attributes (elements, types, or names)"
+                            " in this universe to guess mass from")
+
+        if indices_to_guess is not None:
+            atom_types = atom_types[indices_to_guess] 
+
+        masses = np.array([self.get_atom_mass(atom)
+                           for atom in atom_types], dtype=np.float64)
+        return masses
+
+    def get_atom_mass(self, element):
+        """Return the atomic mass in u for *element*.
+        Masses are looked up in :data:`MDAnalysis.guesser.tables.masses`.
+        .. Warning:: Untill vesion 3.0.0 unknown masses are set to 0.0
```

Comment:
```suggestion
        
        .. Warning:: Until version 3.0.0 unknown masses are set to 0.0
```

This needs a newline to render properly!